compiler/bindings/python/test/api/output_buffer_reference_test.py

```diff
@@ -59,7 +60,7 @@ def run_mmap_free_before_context_test():
     output.write(vmfb_contents)
     mapped_memory = output.map_memory()
     module = VmModule.wrap_buffer(instance, mapped_memory)
-    context = VmContext(instance, modules=[module])
+    loaded_module = load_vm_module(module)
```

Comment:
Nice (I guess :P), this segfaulted on the Linux CI too: https://github.com/iree-org/iree/actions/runs/9490924797/job/26155377341?pr=17657#step:5:2104